dizoo/d4rl/entry/d4rl_ibc_main.py

```diff
@@ -0,0 +1,20 @@
+from ding.entry import serial_pipeline_offline
```

Comment:
why not modify cql and dt

Reply:
`cql` entry changed. To change `dt` entry,  we need to resolve the hardcoded directory first.